scripts/lib/versions.js

```diff
@@ -84,7 +84,7 @@ async function getVersionsJson() {
 }
 
 async function getAnnotations(){
-  const cmd = 'grep -r @NpmPackage ./vaadin*parent/*/src/*/java';
+  const cmd = 'grep -r @NpmPackage\\( ./vaadin*parent/*/src/*/java';
```

Comment:
Maybe even more reliable would be to match start-of-line -> possibly whitespace -> `@NpmPackage`. Then you avoid everything inside comments, even if it says `Use @NpmPackage("foo")`

Reply:
Thought about that, but then a multiline comment could also match.

Reply:
Yeah, I was thinking about javadoc only where you tend to have ` * ` on each comment line